docs/reference/installation.rst

```diff
@@ -2,55 +2,67 @@
 e-commerce Bundles installation
 ===============================
 
-2 installation process are available:
-
-* :ref:`quick-install-ref`
-* :ref:`manual-install-ref`
-
-.. _quick-install-ref:
-
-Quick install
-=============
-
-In case you need to work with Sonata e-commerce for a new project or just want to test it for a simple overview, we strongly advise that you use our demo.
-This demo is available `here <http://demo.sonata-project.org>`_ and the code on `Github <https://github.com/sonata-project/sandbox>`_.
-
-
-See the `README.md <https://github.com/sonata-project/sandbox/blob/2.4-develop/README.md>`_ for the dedicated *but very easy* installation process and `CONTRIBUTING.MD <https://github.com/sonata-project/sandbox/blob/2.4-develop/CONTRIBUTING.md>`_  if you want to contribute.
+Prerequisites
+-------------
+
+PHP 7 and Symfony 2.8, >=3.3 or 4 are needed to make this bundle work, there are
```

Comment:
Symfony 2.8 is not supported anymore and Symfony 3 is only supported form 3.4

Reply:
We don't have platform requirements in our other docs, so I removed it